uwuno/src/test/java/com/learning/uwuno/unit/playerControllerTests.java

```diff
@@ -0,0 +1,133 @@
+package com.learning.uwuno.unit;
+
+import com.learning.uwuno.cards.card;
+import com.learning.uwuno.cards.deck;
+import com.learning.uwuno.controllers.playerController;
+import com.learning.uwuno.player;
+import com.learning.uwuno.services.gameService;
+
+import org.junit.jupiter.api.Test;
+
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.when;
+
+import static org.hamcrest.Matchers.*;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+
+
+@WebMvcTest(playerController.class)
+@AutoConfigureMockMvc
+public class playerControllerTests {
+    @Autowired
+    private MockMvc mvc;
+
+    @Autowired
```

Comment:
Is there a way to do multiple autowired at the same time? If it's too much unnecessary work than dw.